src/select.jsx

```diff
@@ -309,6 +324,8 @@ var classBase = React.createClass({
             onKeyDown={this.onKeyDown}
             onBlur={this.onBlur}
             onClick={this.toggleOpen}
+            onMouseDown={this.props.onMouseDown}
```

Comment:
As far as I know, these aren't safe because you can have `onMouseUp` outside of this element and then it's never called.